distribution/tools/plugin-cli/src/main/java/org/elasticsearch/plugins/InstallPluginAction.java

```diff
@@ -988,7 +1028,7 @@ private static void setFileAttributes(final Path path, final Set<PosixFilePermis
 
     @Override
     public void close() throws IOException {
-        IOUtils.rm(pathsToDeleteOnShutdown.toArray(new Path[pathsToDeleteOnShutdown.size()]));
+        IOUtils.rm(pathsToDeleteOnShutdown.toArray(new Path[0]));
```

Comment:
What was the motivation for this change?

Reply:
I was cleaning up the file based on IDE recommendations.